examples/worldview/webpack.config.js

```diff
@@ -2,7 +2,7 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const webpack = require('webpack');
 const resolve = require('path').resolve;
 
-const config = {
+const CONFIG = {
```

Comment:
Above diff shows config with lower case. Not too sure about the style guide, but should we make all config uppercase?

Reply:
Ah, I suppose any const variable definitions for config should be uppercase. So not all config variables, but this const kind.